src/core/small_string.h

```diff
@@ -55,7 +57,7 @@ class SmallString {
   char prefix_[kPrefLen];
 
   uint32_t small_ptr_;  // 32GB capacity because we ignore 3 lsb bits (i.e. x8).
```

Comment:
@romange still did not understand fully how this should be changed :(

Reply:
no need to change.